api/client/src/main/java/org/projectnessie/client/auth/oauth2/OAuth2Client.java

```diff
@@ -194,9 +195,7 @@ private CompletionStage<Tokens> renewTokens() {
 
   private void log(Tokens tokens, Throwable error) {
     if (error != null) {
-      LOGGER.error("Failed to renew tokens", error);
-    } else {
-      LOGGER.debug("Successfully renewed tokens: {}", tokens);
```

Comment:
Not a good idea to log tokens, they may contain sensitive information.

Reply:
Or maybe only remove the tokens but keep the message?